packages/block-editor/src/components/image-size-control/style.scss

```diff
@@ -1,26 +1,3 @@
 .block-editor-image-size-control {
 	margin-bottom: 1em;
-
-	.block-editor-image-size-control__row {
-		display: flex;
-		justify-content: space-between;
-
-		.block-editor-image-size-control__width,
-		.block-editor-image-size-control__height {
-			margin-bottom: 0.5em;
```

Comment:
Should we keep the margin-bottom override on this one because the block-inspector CSS is adding too much again?

<img width="304" alt="Too much margin on Width control" src="https://user-images.githubusercontent.com/555336/214835982-d21da37d-a5db-4128-9c2e-3741d6adabec.png">

Reply:
Yes, good call!